style.css

```diff
@@ -0,0 +1 @@
+.title-color {color: rgb(119, 160, 37)}
```

Comment:
[OPTIONAL] it would be good to make the whole test pass. You can fix this by running `npx stylelint "**/*.{css,scss}" --fix` in your root directory on the terminal.

Reply:
ok  thank you